helpers.go

```diff
@@ -50,13 +57,90 @@ func GetUniqueTableReferences(node Node) []string {
 	return tableNames
 }
 
+// ValidateTargetTables recursively validates all tables found in the node and return them.
+func ValidateTargetTables(node Node) ([]*ValidatedTable, error) {
```

Comment:
@carsonfarmer lmk if this method would work for you. It returns a list of `ValidatedTable`. IMO it's better than `GetUniqueTableReferences`, because inside `ValidatedTable` you have structured info: 

```
type ValidatedTable struct {
	name    string
	prefix  string
	chainID int64
	tokenID int64
}
```
So, you wouldn't need to do the extracting logic of the chain id on your side.

Reply:
Yeh super nice. Will that ValidateTargetTable function be exposed in sqlparser? That would be great if it was, because then I have a canonical source for testing name structure. I can wrap that and even create a Table reference on the fly for testing etc.

Reply:
Yeah, it will 👍